### ISSUE TITLE
CMakeLists.txt configuration error

### ISSUE THREAD

**Issue by BRoy98** [saantiaguilera/android-api-SecureKeys#44]
I am getting **`-- Configuring incomplete, errors occurred!`** error when compiling with CMake. I used the CMake file from your project. Here is my CMakeLists.txt:

```cmake
cmake_minimum_required(VERSION 3.5)

project(app)

if(ANDROID)
  message (STATUS "Android OS Detected")
else()
  message (STATUS "Not android.")
  find_package(JNI REQUIRED)
endif(ANDROID)

include_directories(${JNI_INCLUDE_DIRS})

include(CheckCXXCompilerFlag)
CHECK_CXX_COMPILER_FLAG("-std=c++11" COMPILER_SUPPORTS_CXX11)

add_subdirectory(core/src/main/cpp)
```

What is missing in this?

**Comment by saantiaguilera** [comment]
You shouldnt use a CMake, Im already providing it by myself (although I know the existing bugs)

I'll be deploying a new version with the fix as soon as I can. Still, you should by no means use your own cmake as I already configure that with the plugin

**Comment by saantiaguilera** [comment]
Refer to: https://github.com/saantiaguilera/android-api-SecureKeys/issues/40

**Comment by saantiaguilera** [comment]
Closing issue, fix should be seen in version: 2.2.0
